### PR TITLE
Test cases

### PR DESCRIPTION
**Pull request by apaelissalde**

### CODE REVIEW

**Review by roxanafalco20 — CHANGES_REQUESTED** [review]
Apa, I have a few comments: I agree with the test cases design. I see that you’re using mostly equivalence partitioning. That’s correct. Those paragraphs are the explanation of how did you think about it. But the design is not the same than a test case. 

Test case structure:
ID | Title | Pre-condition | Description | Steps to reproduce | Expected result

I strongly recommend that you use a table for writing them, that will be easier for you and also for the reader. 

Also, tables will be better for showing the info that you already wrote related to the design (content is accurate, but the way of displaying it is hard to read). 

In summary:
- [x] Use the table that we created on the test case design workshop to write the equivalence partitioning
- [ ] Create a table with the test case structure taking in consideration the previous analysis (equivalence partitioning). Be careful and don’t be redundant in the test cases, remember that you can use a data driven approach.

Please let me know if you need any help on implementing this changes :)

**Review by roxanafalco20 — CHANGES_REQUESTED** [review]
Looks much better! Please add a new column after steps to reproduce which is “Data input” where you can list all the inputs that you mentioned on the steps. 
The idea is that you can have a numbered list of steps with actions (independent from data) and a column specific for the data itself.
Here’s an example:
Screen Shot 2022-03-15 at 17 04 14

**Review by roxanafalco20 — APPROVED** [review]
Green light :)
